docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -102,12 +98,13 @@ for item in $(ceph osd blocklist ls | awk '{print $0}'); do
      ceph osd blocklist rm $item;
 done
 ----
-
++
 [NOTE]
-When a file system client is unresponsive or misbehaving, it may happen that
-the access to the file system is forcibly terminated. This process is called
-eviction. Evicting a CephFS client prevents it from communicating further with MDS daemons and OSD daemons.
-Ordinarily, a blocklisted client may not reconnect to the servers: it must be unmounted and then remounted. However, in some situations it may be useful to permit a client that was evicted to attempt to reconnect. Because CephFS uses the RADOS OSD blocklist to control client eviction, CephFS clients can be permitted to reconnect by removing them from the blocklist.
+====
+When a file system client is unresponsive or misbehaving, the access to the file system might be forcibly terminated. This process is called eviction. Evicting a CephFS client prevents it from communicating further with MDS daemons and OSD daemons.
+
+Ordinarily, a blocklisted client might not reconnect to the servers; you must unmount and then remount the client. However, in some situations, it might be useful to permit a client that was evicted to attempt to reconnect. Because CephFS uses the RADOS OSD blocklist to control client eviction, you can permit CephFS clients to reconnect by removing them from the blocklist.
```

Comment:
```suggestion
Ordinarily, a blocklisted client cannot reconnect to the servers; you must unmount and then remount the client. However, permitting a client that was evicted to attempt to reconnect can be useful. Because CephFS uses the RADOS OSD blocklist to control client eviction, you can permit CephFS clients to reconnect by removing them from the blocklist.
```

Attempting to remove some redundancies